STATUS.md

```diff
@@ -9,17 +9,17 @@
 | NI-DAQmx        | GetNIDAQmxTask(s)                                                       | pins\_to\_nidaqmx\_task(s)                      | test\_nidaqmx                 |
 | NI-DAQmx        | GetAllNIDAQmxTasks                                                      | get\_all\_nidaqmx\_tasks                        | test\_nidaqmx                 |
 | NI-DCPower      | GetNIDCPowerInstrumentNames                                             | get\_all\_nidcpower\_instrument\_names          | test\_nidcpower\_legacy       |
```

Comment:
We removed all the legacy tests because we decided not to support them. We need to update this table to make that clear.

```suggestion
| NI-DCPower      | GetNIDCPowerInstrumentNames                                             | get\_all\_nidcpower\_instrument\_names          | *omitted on purpose       |
```